cookbook/attributes/default.rb

```diff
@@ -17,9 +17,11 @@
 # limitations under the License.
 #
 
+default['apt']['compile_time_update'] = true
+default['build-essential']['compile_time'] = true
 default[:berkshelf_api][:repo]           = "berkshelf/berkshelf-api"
 default[:berkshelf_api][:token]          = nil
-default[:berkshelf_api][:release]        = "v#{Berkshelf::API::Chef.cookbook_version(run_context)}"
+default[:berkshelf_api][:release]        = "v2.1.1"
```

Comment:
need to figure out something better to do here, but right now this lets me test the harness